From linux-kernel-owner@vger.kernel.org Wed May 23 03:41:16 2007
Return-Path:
	<linux-kernel-owner+tglx=40linutronix.de-S1760713AbXEWBkE@vger.kernel.org>
X-Spam-Checker-Version: SpamAssassin 3.1.7-deb (2006-10-05) on debian
X-Spam-Level: 
X-Spam-Status: No, score=0.1 required=5.0 tests=AWL autolearn=unavailable 
	version=3.1.7-deb
Received: from vger.kernel.org (vger.kernel.org [209.132.176.167]) by
	mail.tglx.de (Postfix) with ESMTP id 9CF7665C065 for <tglx@linutronix.de>;
	Wed, 23 May 2007 03:41:16 +0200 (CEST)
Received: (majordomo@vger.kernel.org) by vger.kernel.org via listexpand id
	S1760713AbXEWBkE (ORCPT <rfc822;tglx@linutronix.de>); Tue, 22 May 2007
	21:40:04 -0400
Received: (majordomo@vger.kernel.org) by vger.kernel.org id
	S1757805AbXEWBj4 (ORCPT <rfc822;linux-kernel-outgoing>); Tue, 22 May 2007
	21:39:56 -0400
Received: from rwcrmhc15.comcast.net ([204.127.192.85]:46662 "EHLO
	rwcrmhc15.comcast.net" rhost-flags-OK-OK-OK-OK) by vger.kernel.org with
	ESMTP id S1757432AbXEWBjz (ORCPT <rfc822;Linux-kernel@vger.kernel.org>);
	Tue, 22 May 2007 21:39:55 -0400
Received: from sx.thebigcorporation.com ([69.181.45.228]) by comcast.net
	(rwcrmhc15) with ESMTP id <20070523013954m1500dteb1e>; Wed, 23 May 2007
	01:39:55 +0000
Received: from sx.thebigcorporation.com (localhost.localdomain [127.0.0.1])
	by sx.thebigcorporation.com (8.13.8/8.13.8) with ESMTP id l4N1dsjb030388;
	Tue, 22 May 2007 18:39:54 -0700
Received: (from sven@localhost) by sx.thebigcorporation.com
	(8.13.8/8.13.8/Submit) id l4N1drBv030387; Tue, 22 May 2007 18:39:53 -0700
X-Authentication-Warning: sx.thebigcorporation.com: sven set sender to
	sven@thebigcorporation.com using -f
Subject: [PATCH 2.6.21-rt6]
From:	Sven-Thorsten Dietrich <sven@thebigcorporation.com>
To:	LKML <Linux-kernel@vger.kernel.org>
Cc:	Ingo Molnar <mingo@elte.hu>
Content-Type: text/plain
Organization: The Big Corporation
Date:	Tue, 22 May 2007 18:39:53 -0700
Message-Id: <1179884393.25500.86.camel@sx.thebigcorporation.com>
Mime-Version: 1.0
X-Mailer: Evolution 2.8.3 (2.8.3-2.fc6) 
Sender:	linux-kernel-owner@vger.kernel.org
Precedence: bulk
X-Mailing-List:	linux-kernel@vger.kernel.org
X-Filter-To: .Kernel.LKML
X-Evolution-Source: imap://tglx%40linutronix.de@localhost:8993/
Content-Transfer-Encoding: 8bit

This patch properly exports __spin_lock_irqsave_nested.

signed-off-by: Sven-Thorsten Dietrich <sven@thebigcorporation.com>


---
 kernel/spinlock.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: linux-rt.q/kernel/spinlock.c
===================================================================
--- linux-rt.q.orig/kernel/spinlock.c
+++ linux-rt.q/kernel/spinlock.c
@@ -366,7 +366,7 @@ __spin_lock_irqsave_nested(raw_spinlock_
 #endif
 	return flags;
 }
-EXPORT_SYMBOL(_spin_lock_irqsave_nested);
+EXPORT_SYMBOL(__spin_lock_irqsave_nested);
 
 #endif
 
